CONVENTIONS.md

```
@@ -0,0 +1,20 @@
+팀 컨벤션(전역, 최우선)
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)